micropip/install.py

```diff
@@ -20,6 +20,7 @@ async def install(
     index_urls: list[str] | str | None = None,
     *,
     verbose: bool | int = False,
+    **fetch_kwargs,
```

Comment:
Let's not use kwargs, and pass a dictionary instead.

``
fetch_kwargs: dict[str, Any] | None = {...}
``

Reply:
Also, as `credentials` is part of fetch args, I think we can deprecate it in favor to fetch args.